provider/win_driver_utils.py

```diff
@@ -180,3 +192,31 @@ def copy_file_to_samepath(session, test, params):
         if status != 0:
             test.error("Copy file error,"
                        " the detailed info:\n%s." % output)
+
+
+def memory_leak_check(vm, test, params):
+    """
+    In order to test memory leaks, the driver under test should be
+    uninstalled after driver function test(bz2087102).
```

Comment:
I think it's better to mention that it's related with driver verifier.
e.g.
In order to let the driver verifier to catch memory leaks, driver should be unloaded after driver function.Note that if want to use this function, driver verifier should be enabled before driver function.

Reply:
Updated the annotation.Thanks!